### DISCUSSION TITLE
Is it possible to have a different color scheme per tag

### DISCUSSION THREAD

**Discussion by forte97** [bakkeby/dwm-flexipatch#366] [Q&A]
Hello, is it possible with the patches in this program to set a different color scheme per tag?

**Comment by ghost** [comment]
No, there have been several pull requests adding different forms of the rainbowtags patch. 
Just an idea though, but why not add schemes to tagicons?

**Reply by forte97** [reply]
nevermind I saw it just now, I just patched mine using the pull request.
